src/Webpage/Traits/HasParent.php

```diff
@@ -111,4 +116,25 @@ public function scopeWhereFullPath(Builder $query, string $path)
                 return $item->full_path === str_start($path, '/');
             });
     }
+
+    /**
+     * Get all sub-items of item.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder Sub-items
+     */
+    public function subItems()
+    {
+        return $this->where('parent_id', '=', $this->id)->get();
```

Comment:
Should probably make the `parent_id` definable through a property but default to `parent_id`?

E.g.

```
$table_column = property_exists($this, 'parentColumn`) ? $this->parentColumn : 'parent_id';

return $this->where($table_column, '=', $this->id)->get();
```

Reply:
Good shout @m2de!